src/renderer/utils/file-monitor.test.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jest/prefer-expect-assertions */
+import fs from 'fs';
+
+import { FileMonitor } from './file-monitor';
+let a: [Event, string][];
```

Comment:
If you used `jest.fn()` and `.toHaveBeenCalled()` you won't need this `a` variable or the `c` variable below (https://jestjs.io/docs/en/expect#tohavebeencalled).

Copied from example code:
```typescript
const drink = jest.fn();
drinkAll(drink, 'octopus');
expect(drink).not.toHaveBeenCalled();
```